coforma/swift-tech-challenge
# Issue/apply page

# Title
Issue/apply page
# Summary

This creates the apply page for each college based on their application questions.

Fixes #81, Fixes #88

## Type of change

Please delete options that are not relevant.


# How To Test

1. Load up the project and make sure you are connected to AWS
2. Click on the Apply link in the college cards
3. See the application page! 

Notes: Click through a couple. Some have applications, some have fewer questions than others. They should all render individuall.

# Checklist:
